langlib/lang.error/src/main/ballerina/error.bal

```diff
@@ -15,10 +15,14 @@
 // under the License.
 
 import ballerina/java;
+import ballerina/lang.'value as value;
+
+# The type of value to which `clone` and `cloneReadOnly` can be applied.
+type __Cloneable1 readonly|xml|__Cloneable1[]|map<__Cloneable1>|table<map<__Cloneable1>>;
```

Comment:
There's a duplicate Cloneable definition in the spec for the error module. 

https://github.com/ballerina-platform/ballerina-spec/blob/master/lang/lib/error.bal#L21

Reply:
Yes, but we cannot create a copy because `Cloneable` is already defined as a public type. I am getting redefinition errors.